sdk/core/azure-core/setup.py

```diff
@@ -70,7 +70,6 @@
     python_requires=">=3.8",
     install_requires=[
         "requests>=2.21.0",
-        "six>=1.11.0",
```

Comment:
Add it in the changelog?

Reply:
Yes of course, was curious to see what CI would do, and it was a one click PR from GH UI :p